indico/migrations/versions/20231106_1517_0acf26d68434_add_speakers_can_submit_to_events.py

```diff
@@ -0,0 +1,49 @@
+"""Add speakers_can_submit to events table
+
+Revision ID: 0acf26d68434
+Revises: 31b699664893
+Create Date: 2023-11-06 15:17:38.522399
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '0acf26d68434'
+down_revision = '31b699664893'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('events', sa.Column('speakers_can_submit', sa.Boolean(), nullable=False,
+                  server_default='false'), schema='events')
+    op.execute('''
+        UPDATE events.events ev
+        SET speakers_can_submit = true
+        FROM events.settings es
+        WHERE
+        ev.id = es.event_id AND
+        es.module = 'subcontributions' AND
+        es.name = 'speakers_can_submit' AND
+        es.value::bool;
+    ''')
+    op.execute('''
+        DELETE FROM events.settings es
+        WHERE
+        es.module = 'subcontributions' AND
+        es.name = 'speakers_can_submit';
+    ''')
+    op.alter_column('events', sa.Column('speakers_can_submit', server_default='true'), schema='events')
```

Comment:
```suggestion
    op.alter_column('events', sa.Column('speakers_can_submit', server_default=None), schema='events')
```
Normally we set all `server_defaults` to `None` and rely solely on the default set on the model (if something changes later on it's way easier to debug).
I'd also move it to the line after the `add_column`, most migrations have these next to each other and makes it easier to review.

Reply:
I had this line after the `add_column` and set the `server_default` to `None`, but it was overriding the Models default. So, any new events had `speakers_can_submit` as `False`. For this reason I made this change.